assemblies/core/static/src/main/resources-filtered/spoon.sh

```diff
@@ -191,23 +191,11 @@ case `uname -s` in
 		# linux compatibility packages installed
 	    ARCH=`uname -m`
 		case $ARCH in
-			x86_64)
-				LIBPATH=$CURRENTDIR/../libswt/linux/x86_64/
-				echo "I'm sorry, this FreeBSD platform [$ARCH] is not yet supported!"
-				exit
-				;;
-
 			i[3-6]86)
 				LIBPATH=$CURRENTDIR/../libswt/linux/x86/
 				;;
 
-			ppc)
```

Comment:
Hmmm.... reminds me of my old PowerPC some good 20 years ago...
Why are we removing this? The JIRA case says nothing about removing PPC.

I don't see SWT for PPC being included in the assemblies in https://github.com/pentaho/pentaho-kettle/blob/master/assemblies/client/src/assembly/assembly.xml. Maybe we should remove PPC from all operating systems.